### PR TITLE
docs: translate 'corsinvalidallowmethod' this post

### PR DESCRIPTION
**Pull request by senmu-a**

### CODE REVIEW

**Review by yin1999 — APPROVED** [review]

**Comment by yin1999** [comment]
Hi @senmu-a, we are now using `.md` format, could you please convert this file to markdown. You can see: <https://github.com/mdn/translated-content/blob/main/docs/zh-cn/translation-guide.md> and <https://developer.mozilla.org/zh-CN/docs/MDN/Contribute/Markdown_in_MDN>

**Comment by senmu-a** [comment]
I'm sorry, I'll modify my file tomorrow. Thank you for your review.

File: files/zh-cn/web/http/cors/errors/corsinvalidallowmethod/index.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+translation_of: Web/HTTP/CORS/Errors/CORSInvalidAllowMethod
+---
+{{HTTPSidebar}}
+
+## 原因
+
+```
+原因：CORS 头 'Access-Control-Allow-Methods' 的值无效
+```
+## 哪里错了？
```
Comment (yin1999) [inline files/zh-cn/web/http/cors/errors/corsinvalidallowmethod/index.md:13]:
```suggestion

## 哪里错了？
```

File: files/zh-cn/web/http/cors/errors/corsinvalidallowmethod/index.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+服务器发送的对 {{Glossary("CORS")}} 请求的响应包含 {{HTTPHeader("Access-Control-Allow-Methods")}} 头信息，并且含有至少一个无效的方法名称。
+
+服务器发送 `Access-Control-Allow-Methods` 头信息，让客户端知道它对 CORS 请求支持哪些 [HTTP 请求方法](/zh-CN/docs/Glossary/HTTP)。`Access-Control-Allow-Methods` 的值是以逗号分隔的 HTTP 方法，例如 {{HTTPMethod("GET")}}、{{HTTPMethod("POST")}} 或 {{HTTPMethod("HEAD")}}。如果客户端[用户代理](/zh-CN/docs/Glossary/User_agent)无法识别返回的值，则会发生此错误。
+
+这很有可能只能在服务端解决，方法是修改服务器的配置以不再发送带有 `Access-Control-Allow-Methods` 头的无效或未知方法名称。检查你的用户代理或者你使用的浏览器中的 HTTP 库是最新的。
+
+## 参见
+
+- [HTTP 访问控制（CORS）错误](/zh-CN/docs/Web/HTTP/CORS/Errors)
```
Comment (yin1999) [inline files/zh-cn/web/http/cors/errors/corsinvalidallowmethod/index.md:23]:
```suggestion
- [CORS 错误](/zh-CN/docs/Web/HTTP/CORS/Errors)
```

File: files/zh-cn/web/http/cors/errors/corsinvalidallowmethod/index.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+- Glossary: {{Glossary("CORS")}}
```
Comment (yin1999) [inline files/zh-cn/web/http/cors/errors/corsinvalidallowmethod/index.md:24]:
```suggestion
- 术语：{{Glossary("CORS")}}
```

**Pull request merged**